src/senselab/audio/tasks/text_to_speech/doc.md

```diff
@@ -0,0 +1,60 @@
+# Text to Speech
+
+<button class="tutorial-button" onclick="window.location.href='https://github.com/sensein/senselab/blob/main/tutorials/text_to_speech.ipynb'">Tutorial</button>
+
+## Task Overview
+
+Text-to-speech (TTS) is the task of creating natural-sounding speech from text. This process can be performed in multiple languages and for multiple speakers.
+
+## Models
+
+A variety of models are supported by `senselab` for text-to-speech.
+Each model varies in performance, size, license, language support, and more. Performance may also vary depending, among other reasons, on the length of the text or the target speaker (differences in terms of age, dialects, disfluencies). It is recommended to review the model card for each model before use and refer to the most recent literature for an informed decision.
+
+Several text-to-speech models are currently available through `🤗 Transformers`. These models can be explored on the [Hugging Face Hub](https://huggingface.co/models?library=transformers&pipeline_tag=text-to-speech&sort=downloads).
+**Note**: Some `Hugging Face` models, despite having the `text-to-speech` label on their model cards, may not work with the text-to-speech pipeline. These models are not supported in `senselab`, and identifying them often requires trial and error.
+
+In addition to the models from `🤗 Transformers`, senselab also supports `Mars5-TTS`, which provides text-to-speech capabilities with a target voice (and its transcript) in English.
```

Comment:
It's a little unclear what "[it] provides text-to-speech capabilities with a target voice (and its transcript) in English" means exactly.